app_layout_1.R

```diff
@@ -203,8 +221,17 @@ selected_style=tab_selected_style)))))))
 # app callbacks
 
 # radio button selection options to populate drop down
-
 # this will return something like [{"label": c, "value": c} for c in col_dict[col]]
+app$callback(
+  list(output("radio-dependent", "options")),
+  list(input("radio-selection", "value")),
+  function(chosen_selection){
+
+    outlist <- list(lapply(col_dict[[chosen_selection]], function(x) return(list(label=x, value=x))))
```

Comment:
lets stick with using the `purrr` library - something that can be cleaned up before the layout branch goes into main

Reply:
kk